### DISCUSSION TITLE
Ability to run flow tests in "testing"

### DISCUSSION THREAD

**Discussion by LStokoe** [forcedotcom/salesforcedx-vscode#5371] [Ideas]
**Is your feature request related to a problem? Please describe.**
Feature request - Ability to run flow tests from Code Builder

**Describe what you are ultimately trying to do**
Today, to run flow tests we have to open each flow, and click "run tests" in the top right. I would like to be able to see a list of all flow tests in Code Builder in the "Testing" tab, in the same way that apex tests and LWC tests are listed. I want to be able to easily run flow tests and see the outcome.

**Describe how you think Code Builder can help solve your problem or improve your work**
I want to be able to run flow tests in Code Builder and see the outcome of the test clearly in the UI. This would be helpful for me as a Salesforce Engineer as I can run a number of tests easily when making changes, and would also provide a one-stop-shop for our QA engineer to run multiple tests without having to click in and out of each flow.

**Additional context**
I don't know of any central location to run flow tests, so this would be a really great feature to have and Code Builder seems like a good place to have it

**Comment by smaddox-sf** [comment]
Hi @LStokoe - Thank you for trying out Code Builder and sharing feedback!  I'm going to move this our repo for the Salesforce extension pack so they can consider this feedback for the roadmap.

**Comment by diyer** [comment]
Thank you for the great idea @LStokoe , we are going to transfer this to a discussion to get community feedback
